examples/distro/instrumentation/servlet-3/src/main/java/com/example/javaagent/instrumentation/DemoServlet3Instrumentation.java

```diff
@@ -11,9 +11,9 @@
 import static net.bytebuddy.matcher.ElementMatchers.namedOneOf;
 import static net.bytebuddy.matcher.ElementMatchers.takesArgument;
 
+import io.opentelemetry.javaagent.bootstrap.Java8BytecodeBridge;
```

Comment:
we already have `io.opentelemetry.javaagent.bootstrap` package in `javaagent-bootstrap`, should this be `io.opentelemetry.javaagent.extension.bootstrap`?

Reply:
I thought that we might want to reserve the `io.opentelemetry.javaagent.extension.bootstrap` for something that'll replace `BootstrapPackagesConfigurer`, some SPI that'll be able to append to bootstrap classpath from the extension (I think we've had a request or two for that).
We could rename the `io.opentelemetry.javaagent.extension.bootstrap` package in `javaagent-bootstrap` to `io.opentelemetry.javaagent.extension.bootstrap.internal` (and `javaagent-bootstrap-internal`) -- I've been thinking of doing that for a while, together with renaming `javaagent-tooling` to `javaagent-internal`

Reply:
I'm not totally sure about the `io.opentelemetry.javaagent.bootstrap` package name for the extension classes that lives in the bootstrap class loader, but let's go with it for now at least.

> We could rename the `io.opentelemetry.javaagent.extension.bootstrap` package in `javaagent-bootstrap` to `io.opentelemetry.javaagent.extension.bootstrap.internal` (and `javaagent-bootstrap-internal`) -- I've been thinking of doing that for a while, together with renaming `javaagent-tooling` to `javaagent-internal`

👍